src/angular-bootstrap-toggle.js

```diff
@@ -1,289 +1,313 @@
 (function () {
-    'use strict';
+	'use strict';
+	angular.module('ui.toggle', [])
+		.value('$toggleSuppressError', false)
+		.constant('toggleConfig',
+			{
+				/**
+				 * This object defines supported toggle widget attributes and their default values.
+				 * Angular's ngClick and ngDisabled are handled separately. Search code below.
+				 */
+				/**
+				 * This version simulates checkbox functionality which can have either true or false value.
+				 * User-defined values are not supported.
+				 */
+				'btnCheckboxFalse': false,
+				'btnCheckboxTrue': true,
+				/**
+				 * Type: string/html
+				 * Default: 'On'
+				 * Description: Text of the on toggle
+				 */
+				on: 'On',
+				/**
+				 * Type: string/html
+				 * Default: 'Off'
+				 * Description: Text of the off toggle
+				 */
+				off: 'Off',
+				/**
+				 * Type: string
+				 * Default: ''
+				 * Description: Allows to specify one of the standarg bootstrap's button sizes (class).
+				 * Possible values are btn-lg, btn-sm, btn-xs.
+				 */
+				size: '',
+				/**
+				 * Type: string
+				 * Default: 'btn-primary'
+				 * Description: Class for 'on' state from one of standard bootstrap button types.
+				 * Possible values: btn-default, btn-primary, btn-success, btn-info, btn-warning, btn-danger
+				 */
+				onClass: 'btn-primary',
+				onstyle: '', /* for backward compatibility only */
+				/**
+				 * Type: string
+				 * Default: 'btn-default'
+				 * Description: Class for 'off' state from one of standard bootstrap button types.
+				 * Possible values: btn-default, btn-primary,btn- success, btn-info, btn-warning, btn-danger
+				 */
+				offClass: 'btn-default',
+				offstyle: '', /* for some backward compatibility only */
+				/**
+				 * Type: JSON string
+				 * Default: ''
+				 * Description: Allows to pass user-defined style to the toggle's first immediate child (first DIV inside
+				 * <toggle ...> which is what you actually see as widget's outer container).
+				 * This can be used to alter widget's appearance. Use with caution! Note that 'width' and 'height' values
+				 * will be overwritten by either auto-calculated values or used-specified values from 'width' and 'height'
+				 * attributes.
+				 * Example: <toggle ... toggle-style='{'border': '1px dashed #f00'}'>
```

Comment:
You did a *Replace All* of double-quotes to single-quotes, didn't you? ;-)
Well, in the comment sections, that doesn't work.
On your line 60, the example is not going to work...
So, there are a few places here and there where these changes are not welcome, I'm sorry.

Reply:
yes i did^^. Are we talking about reverting the change form single quoted or are you just saying i have to fix the places where its obvious wrong?

Reply:
Just fix places (in the comments) where it's obviously wrong, the rest is fine, thank you @JPVenson ! ;-)

Reply:
i have made the changes but github does not add them to this pullrequest. do you have any idea why?

Reply:
Never mind ... just delay ;-)